### ISSUE TITLE
Support aggregate functions with more than one input.

### ISSUE THREAD

**Issue by TennyZhuang** [singularity-data/risingwave#2868]
```
ErrorCode::InternalError("Agg with more than 1 input not supported.".into()).into(),
```

e.g. https://www.postgresqltutorial.com/postgresql-aggregate-functions/postgresql-string_agg-function/

**Comment by skyzh** [comment]
I thought we should have already supported string_agg?

**Comment by TennyZhuang** [comment]
> I thought we should have already supported string_agg?

It's broken by new frontend 🤔

**Comment by richardchien** [comment]
Actually only `string_agg` needs more than one argument. So this issue can be closed after #4298 and #4321.
